galaxy_ng/tests/unit/api/base.py

```diff
@@ -95,51 +95,33 @@ def _create_namespace(name, groups=None):
         groups_to_add = {}
         for group in groups:
             groups_to_add[group] = [
-                'galaxy.upload_to_namespace',
-                'galaxy.change_namespace',
-                'galaxy.delete_namespace'
+                'galaxy.namespace_owner',
             ]
         namespace.groups = groups_to_add
         return namespace
 
     @staticmethod
     def _create_partner_engineer_group():
-        pe_perms = [
+        pe_roles = [
             # namespaces
-            'galaxy.add_namespace',
-            'galaxy.change_namespace',
-            'galaxy.upload_to_namespace',
-            'galaxy.delete_namespace',
+            'galaxy.namespace_owner',
 
             # collections
-            'ansible.modify_ansible_repo_content',
-            'ansible.delete_collection',
+            'galaxy.collection_admin',
 
             # users
-            'galaxy.view_user',
-            'galaxy.delete_user',
-            'galaxy.add_user',
-            'galaxy.change_user',
+            'galaxy.user_admin',
 
             # groups
-            'galaxy.view_group',
-            'galaxy.delete_group',
-            'galaxy.add_group',
-            'galaxy.change_group',
+            'galaxy.group_admin',
 
             # synclists
-            'galaxy.delete_synclist',
-            'galaxy.change_synclist',
-            'galaxy.view_synclist',
-            'galaxy.add_synclist',
-
-            # sync config
-            'ansible.change_collectionremote',
+            'galaxy.synclist_owner',
         ]
         pe_group = auth_models.Group.objects.create(
             name='partner-engineers')
 
-        for perm in pe_perms:
-            assign_perm(perm, pe_group)
+        for role in pe_roles:
+            assign_role(role, pe_group)
```

Comment:
We should have the `pe_group` in unit tests & live environments `maintain-pe-group.py` match
* Unit test `pe_group` has various pre-existing roles
* Management command `pe_group` has 1 new role with 14 perms

I assume using pre-existing roles (is the official term "locked"?) is the preferred way to do it

Reply:
I am not seeing `ansible.modify_ansible_repo_content` in any of the locked roles. If we use only locked roles for the pe_group, I guess we would need to add that to a new or existing locked role

Reply:
It seems the unit test `pe_group` has been augmented over time.  I'll correct that and create new users with proper roles for those soon to be failing tests.

`maintain-pe-group.py` is now using locked roles (including the newly created `galaxy.content_admin` locked role)